dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessor.java

```diff
@@ -78,15 +83,29 @@ public void bindTo(StatementExecution cs, int ind) throws SQLException{
         this.cs=cs;
         this.index=ind;    
         if (direction != INPUT){
-            cs.registerOutParameter(ind, getSqlType());
+            cs.registerOutParameter(ind, getSqlType(), direction == RETURN_VALUE);
         }
     }
 
+    private Object transformValue(Object value) throws SQLException {
```

Comment:
OK. Just again - `transformValue` name sounds too general (or not?). If I'm reading the `set` method I may need to visit `transformValue` implementation to understand it's purpose. Can we make it a bit more self-explaining (`transformToJdbcFormat`, `formatForJdbc`, `adaptToJdbc`, `toJdbcFormat` or maybe some better name)? It's a private methods so there is no need to be extremely beautiful.